pro/src/screens/IndividualOffer/Status/StatusToggleButton.tsx

```diff
@@ -12,18 +14,27 @@ export interface StatusToggleButtonProps {
   offerId: number
   isActive: boolean
   status: OfferStatus
-  reloadOffer: () => void
+  setOffer: ((offer: IndividualOffer | null) => void) | null
```

Comment:
(question) pareil ici. Ce n'est pas possible de faire ça ?
```suggestion
  setOffer: ((offer: IndividualOffer) => void) | null
```

Reply:
si tout à fait !